models/job.go

```diff
@@ -338,6 +340,8 @@ type JobService interface {
 	GetAll(context.Context, NamespaceSpec) ([]JobSpec, error)
 	// Delete deletes a job spec from all repos
 	Delete(context.Context, NamespaceSpec, JobSpec) error
+	// BulkDelete deletes bulk job specs
+	BulkDelete(context.Context, NamespaceSpec, []JobSpec, progress.Observer)
```

Comment:
Similar to BulkCreate, this is only used inside the implementing service. Perhaps we don't need to export it.